app/src/main/java/org/wikipedia/views/WikiArticleCardView.kt

```diff
@@ -21,7 +21,7 @@ class WikiArticleCardView constructor(context: Context, attrs: AttributeSet? = n
         View.inflate(context, R.layout.view_wiki_article_card, this)
     }
 
-    fun setTitle(title: String) {
+    fun setTitle(title: String?) {
```

Comment:
Can we keep this as a non-null parameter?

Reply:
This should be a non-null. I agree.